stylesheets/application.css

```diff
@@ -801,6 +797,10 @@ div.description div.wiki {
   line-height: 1.8;
 }
 
+div.description div.wiki .copy-pre-content-link {
+  line-height: normal;
```

Comment:
By setting the font size of this icon to be the same as the default theme, it seems we can eliminate the vertically stretched impression.

```suggestion
  line-height: normal;
  font-size: 0.875rem;
  margin-right: 4px;
```
<img width="660" alt="screenshot 2025-06-03 17 01 14" src="https://github.com/user-attachments/assets/a0e8364c-40ae-4d96-8518-3082510c3c5c" />

Reply:
Thank you for your suggestion.

I have applied your feedback and force-pushed the changes. Looks good!
[https://github.com/redmica/redmine\_theme\_kodomo\_midori/compare/dde6690e52dea8f37ccbacb39abc434e345a4c92..027f8925fc4cbee3083ba0df0f922619f4cd33df](https://github.com/redmica/redmine_theme_kodomo_midori/compare/dde6690e52dea8f37ccbacb39abc434e345a4c92..027f8925fc4cbee3083ba0df0f922619f4cd33df)
Additionally, I have applied the style not only to the copy button for issue descriptions, but to all copy buttons.

Issue
![CleanShot 2025-06-04 at 11 48 02](https://github.com/user-attachments/assets/4c12f5d5-ff60-4b0c-b8b0-4b70364303d0)

Issue notes
![CleanShot 2025-06-04 at 11 52 37](https://github.com/user-attachments/assets/4cd5f1f7-aa14-4e07-b298-a76f6c64f0ed)

News
![CleanShot 2025-06-04 at 11 52 07](https://github.com/user-attachments/assets/bb3ffa3b-e231-4f35-95e5-5938d1c33f86)

Reply:
Additionally, I have added a style to adjust the position of the copy button in forum topics.
[https://github.com/redmica/redmine\_theme\_kodomo\_midori/compare/027f8925fc4cbee3083ba0df0f922619f4cd33df..83a92c0cbdc4550407b26cbc2b56a1df60cf10a4](https://github.com/redmica/redmine_theme_kodomo_midori/compare/027f8925fc4cbee3083ba0df0f922619f4cd33df..83a92c0cbdc4550407b26cbc2b56a1df60cf10a4)

Before
![CleanShot 2025-06-04 at 12 11 54](https://github.com/user-attachments/assets/437f7595-d5f0-4eba-85cb-3ce434bfd57e)

After
![CleanShot 2025-06-04 at 12 11 27](https://github.com/user-attachments/assets/6622eecf-35ac-4bb5-9879-52e5a2652979)